salt/environment_settings.yml

```diff
@@ -716,6 +716,13 @@ environments:
         - fluentd
         - rabbitmq
       rds:
+        - name: mitxpro
+          multi_az: True
+          engine: postgres
+          vault_plugin: postgresql-database-plugin
+          db_instance_class: db.t2.medium
```

Comment:
I would steer clear of any t2 instance class. Probably worth just going with an M5